src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -1989,6 +1995,53 @@ public String registerDataset() {
         FacesContext.getCurrentInstance().addMessage(null, message);
         return returnToDatasetOnly();
     }
+    
+    public String updateCurrentVersion() {
```

Comment:
@qqmyers can you add comments here and in the api that this code is duplicated so that we know thatif a change occurs we would need to update in multiple places (until we move to an onSuccess)?

Reply:
added comments and brought the PR back up to date with dev...